bin/real-wazo-upgrade

```diff
@@ -107,6 +107,17 @@ upgrade() {
 
 upgrade_dahdi_linux_modules(){
 	VERSION=$1
+
+	if ! grep -q "^ii dahdi$" <(dpkg -l | awk '{ print $1 " " $2 }'); then
+		# dahdi is not installed
+		return
+	fi
+
+	if grep -q "^ii.*dahdi-linux-modules-$(uname -r)" <(dpkg -l); then
```

Comment:
Use the same fonction everywhere https://github.com/wazo-platform/wazo-upgrade/blob/master/pre-start.d/40-remove-unused-packages.sh#L9
This is language independent